pkg/ovs/ovsconfig/ovs_client.go

```diff
@@ -783,12 +787,30 @@ func (br *OVSBridge) GetOVSVersion() (string, Error) {
 		klog.Error("Transaction failed: ", err)
 		return "", NewTransactionError(err, temporary)
 	}
+
 	if len(res[0].Rows) == 0 {
-		klog.Warning("Could not find ovs_version")
-		return "", nil
+		klog.Warning("Could not find ovs_version in the OVS query result")
+		return "", NewTransactionError(err, false)
+	} else {
+		return ParseOvsVersion(res[0].Rows)
 	}
+}
 
-	return res[0].Rows[0].(map[string]interface{})["ovs_version"].(string), nil
+func ParseOvsVersion(ii interface{}) (string, Error) {
+	helpMessage := "ovs-vsctl --no-wait set Open_vSwitch . ovs_version=$OVS_VERSION"
```

Comment:
The help commands `ovs-vsctl --no-wait set Open_vSwitch . ovs_version=$OVS_VERSION` is only work for Windows platform.

And for Linux there's no help command.

You can move the variable `helpMessage` to  pkg/ovs/ovsconfig/default_windows.go as a const value.

Reply:
ok cool